core/ui/EditorToolbar/picture-dropdown.tid

```diff
@@ -4,8 +4,44 @@ title: $:/core/ui/EditorToolbar/picture-dropdown
 [img[$(imageTitle)$]]
 \end
 
+\define insert-external-image()
```

Comment:
@duarteframos, feel free to delete your comment. It was all my misunderstanding. ;-)